R/images.R

```diff
@@ -63,7 +63,7 @@ capturePlot <- function(expr, filename = tempfile(fileext = ".png"),
   if (!"..." %in% argnms) {
     # Only include `width`, `height`, and `res` if the corresponding formal
     # parameters are present.
-    args <- args[names(args) %in% argnms | names(args) == "filename"]
+    args <- args[names(args) %in% argnms]
   }
   args <- c(list(filename = filename), args, rlang::list2(...))
```

Comment:
I don't think we can assume that most/all devices will have a filename argument (`svglite::svglite()` is one such example that uses `file` instead of `filename` so perhaps we should do some sort of partial matching on `file*`)